src/History.tsx

```diff
@@ -64,7 +64,12 @@ class HistoryContainer extends React.Component<Props, {}> {
                 sendPostBack(this.props.botConnection, value, this.props.user, this.props.format.locale);
                 break;
 
+            case "call":
             case "openUrl":
+            case "playAudio":
+            case "playVideo":
+            case "showImage":
+            case "downloadFile":
             case "signin":
                 window.open(value);
```

Comment:
According to window.open docs, it always opens in a new window (or tab, I suppose):
http://www.w3schools.com/jsref/met_win_open.asp
"The open() method opens a new browser window.